tests/modules/asset_groups/resources/test_patch_asset_group.py

```diff
@@ -116,3 +117,73 @@ def test_patch_asset_group(
             )
 
         tus_utils.cleanup_tus_dir(transaction_id)
+
+
+# similar to the above but against the AGS-as-sighting endpoint
+def test_patch_asset_group_sighting_as_sighting(
+    flask_app_client, researcher_1, regular_user, test_root, db, empty_individual
+):
+    # pylint: disable=invalid-name
+    from tests.modules.asset_groups.resources.utils import TestCreationData
+    from tests import utils
+
+    transaction_id, test_filename = tus_utils.prep_tus_dir(test_root)
+    asset_group_uuid = None
+    try:
+        data = TestCreationData(transaction_id)
+        data.add_filename(0, test_filename)
+        response = asset_group_utils.create_asset_group(
+            flask_app_client, regular_user, data.get()
+        )
+        asset_group_uuid = response.json['guid']
+        asset_group_sighting_guid = response.json['asset_group_sightings'][0]['guid']
+
+        # Regular user can create it but not read it??????
+        asset_group_utils.read_asset_group_sighting_as_sighting(
+            flask_app_client, regular_user, asset_group_sighting_guid, 403
+        )
+
+        # Researcher should be able to
+        group_sighting = asset_group_utils.read_asset_group_sighting_as_sighting(
+            flask_app_client, researcher_1, asset_group_sighting_guid
+        )
+        from app.modules.asset_groups.schemas import SIGHTING_FIELDS_IN_AGS_CONFIG
+
+        expected_fields = {'guid', 'stage', 'completion', 'assets'}
+        expected_fields.update(SIGHTING_FIELDS_IN_AGS_CONFIG)
```

Comment:
We can't expect all the fields in `SIGHTING_FIELDS_IN_AGS_CONFIG` to be there anymore because we're removing them if their values are None